Source/WebCore/css/parser/CSSSelectorParser.cpp

```diff
@@ -884,6 +884,12 @@ std::unique_ptr<MutableCSSSelector> CSSSelectorParser::consumePseudo(CSSParserTo
             block.consumeWhitespace();
             if (!innerSelector || !block.atEnd())
                 return nullptr;
+            if (innerSelector->selector() && innerSelector->selector()->selectorList()) {
+                for (const auto& subSelector : *innerSelector->selector()->selectorList()) {
+                    if (subSelector.tagHistory())
```

Comment:
Doesn't this also make compound selector invalid (like `.a.b`) ?

Reply:
@AlexsanderDamaceno Yes, you should update the test. Make sure you also update it in the WPT repository though. Wrt `.a.b`, make sure that this is tested somewhere also.